.drone.yml

```diff
@@ -21,16 +21,16 @@ pipeline:
     secrets: [gh_token]
     commands: node scripts/failcheck.js
     when:
-      status: [ success, failure ]
+      status: [success, failure]
 
   deploy-twigjs-preview:
     group: previews
     image: node:dubnium
     secrets: [gh_token, netlify_auth_token, netlify_site_id]
     commands:
-      - yarn clean
-      - yarn dist
-      - 'yarn netlify deploy --site $${NETLIFY_SITE_ID} --message "Drone build: ${CI_BUILD_NUMBER}"'
+      - yarn twing:clean
+      - yarn twing:dist
+      - 'yarn netlify deploy --site $${NETLIFY_SITE_ID} --alias ${DRONE_COMMIT_BRANCH} --message "Drone build: ${CI_BUILD_NUMBER} for the branch: ${DRONE_COMMIT_BRANCH}"'
```

Comment:
I will setup drone locally for the ecl-twig project in a bit to have it for future updates too.
Although I haven't tested locally yet, this change looks problematic. This deployment message is used by the script https://github.com/ec-europa/ecl-twig/blob/master/scripts/set-netlify-deployment-status.js#L58 to find the specific deployment. It seems to me the current deployment will not be found and the status will not be updated.